### ISSUE TITLE
Combining 2 images into 1 side by side image

### ISSUE THREAD

**Issue by james-Solutions** [lovell/sharp#3137]
Hello,

I am using Sharp to create thumbnails from existing images, loading pngs into nodebuffers that is. This works great image by image. However, there are times I may want 2 images to be combined in a way that the images are next to each other. Not overlayed each other. Meaning, say I have images A and B. I would want to combine both images and resize them to output like:

|--------------------|
|      A      |      B       |
|--------------------|
https://imgur.com/ci5umRa

The input images maybe any size, but I want the combined images to be 96px in width. Which works with resize, however, I am unsure how to make the images side by side rather than overlayed.

**Comment by lovell** [comment]
Please see future possible enhancement #1580

**Comment by lovell** [comment]
Yes, it's possible to replicate this feature using the [composite](https://sharp.pixelplumbing.com/api-composite) operation, which is what the `join-images` package appears to do. A native join feature should be much faster.
